packages/calcite-components/src/components/popover/popover.e2e.ts

```diff
@@ -729,6 +729,54 @@ describe("calcite-popover", () => {
     });
   });
 
+  describe("warning messages", () => {
```

Comment:
I'll submit a follow-up PR to clean this up and also try to add some reusable tests/utils.